src/mokka/src/main/java/pl/hycom/mokka/service/openapi/DefaultOpenApiSchemaParsingStrategy.java

```diff
@@ -0,0 +1,18 @@
+package pl.hycom.mokka.service.openapi;
+
+import io.swagger.v3.oas.models.Components;
+import io.swagger.v3.oas.models.media.Schema;
+import org.springframework.stereotype.Component;
+
+/**
+ * @author Kamil Adamiec (kamil.adamiec@hycom.pl)
+ */
+@Component
```

Comment:
Please remove the annotation. As for now we can pass this implementation directly to related converter via constructor when instantiating in `ConversionConfiguration`